packages/uzbekistan/README.md

```diff
@@ -0,0 +1,49 @@
+# 🇺🇿 Uzbekistan
+
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-blue.svg)](https://creativecommons.org/licenses/by/4.0/)
+
+## Contents:
+
+### All Regions:
+* Andijon
+* Buxoro
+* Farg‘ona
+* Jizzax
+* Namangan
+* Navoiy
+* Qashqadaryo
+* Samarqand
+* Sirdaryo
+* Surxondaryo
+* Toshkent
+* Xorazm
+
+### Sea area:
+* Orol Dengizi
+
+### Autonomous Republic:
+* Qoraqalpog‘iston
```

Comment:
Could you use [English names](https://en.wikipedia.org/wiki/Regions_of_Uzbekistan) (in the SVG too) please?

Reply:
Done